tests/integration/customizations/test_generatecliskeleton.py

```diff
@@ -15,14 +15,14 @@
 import logging
 
 import mock
-from nose.tools import assert_equal
+import pytest
 import ruamel.yaml as yaml
 
-from awscli.testutils import capture_output
 from awscli.clidriver import create_clidriver
 
 
-def test_can_generate_skeletons_for_all_service_comands():
+def get_all_cli_skeleton_commands():
+    skeleton_commands = []
     environ = {
```

Comment:
What do you think about moving environment patching to a separate fixture.

Reply:
So I looked into making this a separate fixture, but one thing I could not determine is if you can apply a fixture to something that is not declared as a fixture or being ran as part of the test suite. Do you know if it is possible to call the fixture inside of this function (which is neither a fixture or test case itself)?

Looking over the diff again, I many need to create a fixture anyway because I'm no longer patching the environment when I'm running the generate skeleton command, but unfortunately seems that I will have to duplicate the logic in two places and cannot just reuse it as a fixture.